Update advanced-spam-filtering-asf-options.md (#754)
MicrosoftDocs/microsoft-365-docs: microsoft-365/security/office-365-security/advanced-spam-filtering-asf-options.md

# Advanced spam filtering options

> >[!NOTE]

Reviewer: ```suggestion
> [!NOTE]
```

> > The Anti-spam policy Advanced Spam filter settings are being deprecated at the time of this writing. Our recommended settings for these are to turn them **OFF**. The functions that were available in the Advanced Spam filter is being incorporated into the other parts of the filtering stack.

Reviewer: ```suggestion
> The Advanced Spam filter settings of the Anti-spam policy are currently being deprecated. Our recommended settings for these are to turn them **OFF**. The functions that were available in the Advanced Spam filter is being incorporated into other parts of the filtering stack.
```
